.travis.yml

```diff
@@ -1,9 +1,10 @@
 sudo: false
 language: java
 jdk:
```

Comment:
Instead of the `jdk` tag, maybe consider a build matrix like:

```yaml
matrix:
  include:
    - jdk: openjdk8
    - jdk: oraclejdk8
    - jdk: openjdk9
    - jdk: oraclejdk9
    - jdk: openjdk10
    - jdk: oraclejdk10
    - jdk: openjdk11
    - os: osx
      osx_image: xcode10
```